capstone/src/test/java/com/google/sps/objects/PostTest.java

```diff
@@ -157,7 +158,7 @@ public void removeLikeMultipleTest() {
   }
 
   @Test
-  public void getAndCreatePostEntityTest() {
+  public void toAndFromPostEntityTest() {
```

Comment:
I know Himani was having some issues with this test because some data needed to be stored in the DB to be formatted correctly, so double check this is working as intended and passing. If it isn't, you likely just need to follow what she did for the toFromEntityTest in Objects/UserTest.java